extensions/common/context.cc

```diff
@@ -293,9 +293,13 @@ namespace {
 bool extractPeerMetadataFromUpstreamMetadata(const std::string& metadata_type,
                                              flatbuffers::FlatBufferBuilder& fbb) {
   std::string endpoint_labels;
+  std::string identity;
```

Comment:
Undo these changes?

Reply:
Good catch; this was from an older implementation